src/environment/DHScreening.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * MRChem, a numerical real-space code for molecular electronic structure
+ * calculations within the self-consistent field (SCF) approximations of quantum
+ * chemistry (Hartree-Fock and Density Functional Theory).
+ * Copyright (C) 2023 Stig Rune Jensen, Luca Frediani, Peter Wind and contributors.
+ *
+ * This file is part of MRChem.
+ *
+ * MRChem is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MRChem is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with MRChem.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * For information on the complete list of contributors to MRChem, see:
+ * <https://mrchem.readthedocs.io/>
+ */
+
+#pragma once
+
+#include "Cavity.h"
+#include "utils/print_utils.h"
+#include <MRCPP/MWFunctions>
+#include <MRCPP/Printer>
+
+namespace mrchem {
+/** @class DHScreening
+ *
+ * @brief Square of the Debye-Huckel Screening parameter.
+ * TODO write proper docs for this function
+ *
+ * Here the \f$\bar{\kappa}\f$ is dependent on a separate cavity function with its own radius.
+ *
+ */
+
+class Cavity;
+
+class DHScreening final : public mrcpp::RepresentableFunction<3> {
+public:
+    /** @brief Standard constructor. Initializes the #cavity_ion and #kappa_out with the input parameters.
+     *  @param cavity_ion interlocking spheres of Cavity class.
+     *  @param kappa_out value of the screening function outside the #cavity_ion.
+     *  @param formulation Decides which formulation of the #DHScreening function to implement, only continuous screening function available
+     * available as of now.
+     */
+    DHScreening(const Cavity cavity_ion, double kappa_out, std::string formulation);
```

Comment:
Pass large objects which you're not going to modify by `const` reference https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-in
```suggestion
    DHScreening(const Cavity & cavity_ion, double kappa_out, const std::string & formulation);
```